convert2rhel/unit_tests/actions/conversion/preserve_only_rhel_kernel_test.py

```diff
@@ -0,0 +1,434 @@
+# Copyright(C) 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+import glob
+import os
+import re
+
+import pytest
+import six
+
+from convert2rhel import actions, pkghandler, unit_tests, utils
+from convert2rhel.actions.conversion import preserve_only_rhel_kernel
+from convert2rhel.systeminfo import Version, system_info
+from convert2rhel.unit_tests import (
+    GetInstalledPkgsWDifferentFingerprintMocked,
+    RunSubprocessMocked,
+    StoreContentToFileMocked,
+)
+from convert2rhel.unit_tests.conftest import centos7
+
```

Comment:
```suggestion
from convert2rhel import actions, pkghandler, unit_tests, utils

from convert2rhel.actions.conversion import preserve_only_rhel_kernel
from convert2rhel.systeminfo import Version, system_info
from convert2rhel.unit_tests import (
    GetInstalledPkgsWDifferentFingerprintMocked,
    RunSubprocessMocked,
    StoreContentToFileMocked,
)
from convert2rhel.unit_tests.conftest import centos7
```

Seems to not be used